BrainPortal/app/models/portal_task.rb

```diff
@@ -637,7 +645,9 @@ def include?(paramspath) #:nodoc:
     end
 
     def keys #:nodoc:
-      @real_errors.keys
+      display_keys = []
+      @real_errors.each {|err| display_keys << @base.class.human_attribute_name(err) }
```

Comment:
This won't work I think, the each() method of the real_errors objects expects a block with two arguments; if you're using a single argument, it will be set to a pair.